app/build.gradle

```diff
@@ -79,7 +79,7 @@ configurations.all { config ->
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.1.0-alpha01'
+    implementation 'androidx.appcompat:appcompat:1.1.0-alpha02'
```

Comment:
Upgrade was breaking linter. Is this ok to upgrade?

Reply:
yeah I had to as well.